packages/svelte-undp-design/src/lib/Table/Table.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  export let variant: "default" | "striped" | "border" = "default";
+  export let size: "small" | "large" = "small";
+
+  export let data :Array<object> = [];
```

Comment:
Could you try to avoid using `object`?

maybe like `{[key: string]: string | number | HTMLElement | what else?}`